src/routes/wohnen/containers/WohnenExtra.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import * as React from 'react'
+
+import Spinner from 'react-spinkit'
+import Helmet from 'react-helmet'
+import type { StateType } from 'modules/app/StateType'
+import { connect } from 'react-redux'
+import WohnenOfferModel from '../../../modules/endpoint/models/WohnenOfferModel'
+import CityModel from '../../../modules/endpoint/models/CityModel'
+import ExtraModel from '../../../modules/endpoint/models/ExtraModel'
+import OfferList from '../components/OfferList'
+
+type PropsType = {
+  offers: Array<WohnenOfferModel>,
+  city: string,
+  language: string,
+  extras: Array<ExtraModel>,
+  cities: Array<CityModel>
+}
+
+class WohnenExtra extends React.Component<PropsType> {
+  render () {
+    const LoadingSpinner = () => <Spinner name='line-scale-party' />
```

Comment:
This shouldn't be defined inside the render method...
Either put it outside the WohnenExtra class as a constant or just use `<Spinner name='line-scale-party' />` down below :)

Reply:
This is fixed in another task.

Reply:
I think we are using LoadingSpinner like this quite sometime, so it would maybe make sense to create a separate component for this.